akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/CacheDirective.scala

```diff
@@ -96,12 +96,12 @@ object CacheDirectives {
 
   // http://tools.ietf.org/html/rfc7234#section-5.2.2.6
   final case class `private`(fieldNames: immutable.Seq[String]) extends FieldNamesDirective with ResponseDirective
-  object `private` {
-    @varargs def apply(fieldNames: String*): `private` = apply(immutable.Seq(fieldNames: _*))
+  case object `private` {
```

Comment:
`case object` is not strictly needed for `private`. In the `no-cache` case the companion object can actually be used as a standalone value.